projectroles/tests/test_commands.py

```diff
@@ -601,10 +619,45 @@ def test_command_cleanappsettings(self):
                     CLEAN_LOG_PREFIX + START_MSG,
                     (
                         CLEAN_LOG_PREFIX
-                        + 'Deleting "settings.example_project_app.ghost" '
-                        'from project "{}"'.format(self.project.title)
+                        + CLEAN_LOG_DELETE_GHOST
+                        + ' "{}": '.format(self.project.title)
+                        + CLEAN_LOG_DELETE_GHOST_END
                     ),
                     CLEAN_LOG_PREFIX + END_MSG,
                 ],
             )
         self.assertEqual(AppSetting.objects.count(), 5)
+        self.assertQuerysetEqual(
+            AppSetting.objects.filter(name='ghost'),
+            [],
+        )
+
+    def test_command_cleanappsettings_project_types(self):
```

Comment:
I think we can drop the `cleanappsettings` from the test method names here since we have a separate class for the command anyway. Just call them `test_command_*()`.

(I know, this wasn't originally decided by you but someone else working on this before, but let's clean things up while we're at it :))